hw1/task1/matrices.py

```diff
@@ -0,0 +1,37 @@
+from vectors import *
+import copy
+
+
+class Matrix:
+    content: List[Vector]
+
+    def __init__(self, first: Vector, *args: Vector):
+        if not all(len(first) == len(vector) for vector in args):
+            raise TypeError("Rows must be the same length")
+        self.content = [first, *args]
+
+    def __str__(self):
+        return str(self.content)
+
+    def __add__(self, other):
+        return Matrix(*[get_vector_sum(x, y) for x, y in zip(self.content, other.content)])
+
+    def get_dimensions(self):
+        return {"columns": len(self.content), "rows": len(self.content[0])}
```

Comment:
Actually, it is the other way around. Currently, you have:

`a = Matrix([1, 2, 3, 4],`
`           [5, 6, 7, 8],`
`           [9, 10, 11, 12])`
`print(a.get_dimensions())`

`{'columns': 3, 'rows': 4}`